src/components/quiz-radio/quiz-radio.module.scss

```
@@ -0,0 +1,293 @@
+
+/**
+ * General purpose SVG logic
+ */
+.icon {
+	flex-shrink: 0;
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: var(--quiz-option_icon_size);
+	height: var(--quiz-option_icon_size);
+	// TODO: Double-check with Ed?
```

Stylelint (17.3.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)